docs/resources/trusted_key.md

```diff
@@ -0,0 +1,34 @@
+---
+subcategory: "Security"
+---
+# Artifactory Trusted Key Resource
+
+Provides an Artifactory Trusted Key resource. This can be used to create and manage Artifactory Trusted Keys.
```

Comment:
Please add links to the [API documentation](https://jfrog.com/help/r/jfrog-rest-apis/set-distributionpublic-gpg-key) and to the [UI documentation](https://jfrog.com/help/r/jfrog-platform-administration-documentation/managing-public-keys).